Command/SmokeTestRunCommand.php

```diff
@@ -5,6 +5,7 @@
 use Smartbox\CoreBundle\Utils\SmokeTest\Output\SmokeTestOutputInterface;
 use Smartbox\CoreBundle\Utils\SmokeTest\Output\SmokeTestOutputMessage;
 use Smartbox\CoreBundle\Utils\SmokeTest\SmokeTestInterface;
+use Smartbox\Integration\FrameworkBundle\Tools\SmokeTests\CanCheckConnectivityInterface;
```

Comment:
Core bundle doesn't have as dependency framework bundle. This means that if there were tests for this command it would fail. If there are changes in a class, you should provide tests for that changes